mantle/platform/metal.go

```diff
@@ -575,6 +570,13 @@ func (inst *Install) InstallViaISOEmbed(kargs []string, liveIgnition, targetIgni
 		insecureOpt = "--insecure"
 	}
 	pointerIgnitionPath := "/var/opt/pointer.ign"
+
+	targetDevice := "/dev/vda"
+	// For aarch64, the cdrom is a pci blk device /dev/vda
```

Comment:
Fine for now, note for future: I think we should set things up using the `serial=` identifer and use `/dev/disk/by/id` in the future, see also https://github.com/coreos/coreos-assembler/pull/1343

Reply:
noted it.